src/oc/fileio.cpp

```diff
@@ -13,6 +18,14 @@
 #include "hocparse.h"
 #include <errno.h>
 #include "nrnfilewrap.h"
+#include <fmt/format.h>
+
+// separator for items in env variables
+#if defined(WIN32)
+static const auto os_pathsep = std::string(";");
+#else
+static const auto os_pathsep = std::string(":");
+#endif
```

Comment:
Not sure if the scope of this var should be placed at the level of `split_paths` (see below) or left as-is at file level scope.